apps/web/components/Charts/v2/renderers/ChartRenderer.tsx

```diff
@@ -39,6 +39,64 @@ type ChartProps = {
   slice: IChart
 }
 
+const transformData = (items: any) => {
+  return {
+    statisticsForHeader: items
+      .flatMap((item: { categoryItems: any }) => item.categoryItems)
+      .map((item: { id: string; value: string }) => ({
+        key: item.id,
+        value: parseInt(item.value),
+      })),
+  }
+}
+
+const getData = (slice: IChart, queryResult: DataItem[]): any => {
+  const base = slice.sourceData ? JSON.parse(slice.sourceData) : queryResult
+
+  const values = slice.components.map((component) =>
+    JSON.parse(component.values ?? '{}'),
```

Comment:
Can we use || instead of ?? to prevent potential runtime errors in the future if values is an empty string